lib/logger.js

```diff
@@ -3,7 +3,7 @@ var winston = require('winston');
 var logger = new (winston.Logger)({
     transports: [
       new (winston.transports.Console)({
-          level: 'info',
+          level: 'error',
```

Comment:
For debugging of big repos which are run for coverage message produce more noise than useful info, plus probably some time for printing required